.github/workflows/high-depends.yml

```diff
@@ -14,7 +14,7 @@ jobs:
             fail-fast: false
             matrix:
                 os: [ubuntu-latest, windows-2019]
-                node-versions: ['18', '20', '21', '22']
+                node-versions: ['18', '20', '22']
```

Comment:
should we run tests on node 23 ?

Reply:
Yes, but it looks like it will be available in ~october 2024

Reply:
Let's merge, I've opened #1323 so we won't forget